CHANGELOG.md

```diff
@@ -1,3 +1,108 @@
+  but should only be used if it is necessary to add multiple accounts at once,
+  such as when restoring a wallet from seed where multiple accounts had been
+  previously derived.
+
+Key derivation API:
+- `zcashlc_derive_spending_key`
+- `zcashlc_spending_key_to_full_viewing_key`
+
+Address retrieval, derivation, and verification API:
+- `zcashlc_get_current_address`
+- `zcashlc_get_next_available_address`
+- `zcashlc_get_sapling_receiver_for_unified_address`
+- `zcashlc_get_transparent_receiver_for_unified_address`
+- `zcashlc_is_valid_unified_address`
+- `zcashlc_is_valid_unified_full_viewing_key`
+- `zcashlc_list_transparent_receivers`
+- `zcashlc_get_typecodes_for_unified_address_receivers`
+- `zcashlc_free_typecodes`
+- `zcashlc_get_address_metadata`
+Balance API:
+- `zcashlc_get_verified_transparent_balance_for_account`
+- `zcashlc_get_total_transparent_balance_for_account`
+
+New memo access API:
+- `zcashlc_get_received_memo`
+- `zcashlc_get_sent_memo`
+
+## Changed
+- `zcashlc_create_to_address` now has been changed as follows:
+  - it no longer takes the string encoding of a Sapling extended spending key
+    as spend authority; instead, it takes the binary encoded form of a unified
+    spending key as returned by `zcashlc_create_account` or 
+    `zcashlc_derive_spending_key`. See the note above.
+  - it now takes the minimum number of confirmations used to filter notes to
+    spend as an argument.
+  - the memo argument is now passed as a potentially-null pointer to an
+    `[u8; 512]` instead of a C string.
+- `zcashlc_shield_funds` has been changed as follows:
+  - it no longer takes the transparent spending key for a single P2PKH address
+    as spend authority; instead, it takes the binary encoded form of a unified
+    spending key as returned by `zcashlc_create_account`
+    or `zcashlc_derive_spending_key`. See the note above.
+  - the memo argument is now passed as a potentially-null pointer to an
+    `[u8; 512]` instead of a C string.
+  - it no longer takes a destination address; instead, the internal shielding
+    address is automatically derived from the account ID.
+- Various changes have been made to correctly implement ZIP 316:
+  - `FFIUnifiedViewingKey` now stores an account ID and the encoding of a
+    ZIP 316 Unified Full Viewing Key.
+  - `zcashlc_init_accounts_table_with_keys` now takes a slice of ZIP 316 UFVKs.
+- `zcashlc_put_utxo` no longer has an `address_str` argument (the address is
+  instead inferred from the script).
+- `zcashlc_get_verified_balance` now takes the minimum number of confirmations
+  used to filter received notes as an argument.
+- `zcashlc_get_verified_transparent_balance` now takes the minimum number of
+  confirmations used to filter received notes as an argument.
+- `zcashlc_get_total_transparent_balance` now returns a balance that includes
+  all UTXOs up to those in the latest block (i.e. those with 0 confirmations.)
```

Comment:
"0 confirmations" normally means mempool transactions, but "up to those in the latest block" suggests otherwise (i.e. 1 confirmation or more). Please clarify.

Reply:
The code is certainly doing `0` confirmations, and that does look like it includes mempool tx, as you say.

Since @str4d had commented on that code earlier and said “with a confirmations of `0`, the two should be identical,” I’m going to assume that means the mempool tx are intentional and update the changelog.